src/lightning_app/utilities/packaging/cloud_compute.py

```diff
@@ -1,6 +1,14 @@
 from dataclasses import asdict, dataclass
 from typing import List, Optional, Union
 
+_name_to_devices_map = {
+    "default": 1,
+    "cpu": 1,
+    "gpu": 1,
+    "gpu-fast": 1,
+    "gpu-fast-multi": 4,
+}
+
```

Comment:
Could you add explain how this map is built ?
Also, I remember that types of machines e.g. `default, gpu-fast ..... ` are not hardcoded in the framework for a reason. cc: @awaelchli

Reply:
This is simply the associated number of GPUS for each one of those instances. Right now, this is mainly for there for simplicity but `auto` devices should do the trick.

Reply:
Removed.